### ISSUE TITLE
Smooth loss plots

### ISSUE THREAD

**Issue by shenoynikhil** [torchmd/torchmd-net#166]
Hi,

I was curious on how you guys achieve smooth loss plots for QM9. If I just port over this model and use a simple Adam optimizer to reduce the L1 loss on QM9 targets, the loss curves are quite jittery. Any practical tips that I could make use of to get smooth loss curves?

**Comment by peastman** [comment]
Are you looking at training loss or validation loss?  The former is generally much smoother than the latter.

**Comment by shenoynikhil** [comment]
Training loss (per step not epoch)

**Comment by peastman** [comment]
That's expected to be very noisy.  Every batch is independent.  The loss on one batch may be very different from the loss on the next batch.  Using a larger batch size will reduce the variation somewhat.

**Comment by shenoynikhil** [comment]
I am not sure, but can you visually say this looks alright,
<img width="434" alt="Screenshot 2023-03-23 at 3 08 45 PM" src="https://user-images.githubusercontent.com/28651490/227375927-41434c75-2971-4698-a24a-0d3ea2f3449e.png">

**Comment by peastman** [comment]
It looks plausible.  It certainly seems to be learning.  You probably want to plot it on a log scale so you can see what happens as the loss gets smaller.

**Comment by shenoynikhil** [comment]
Thanks a lot!
